examples/data/mysql/src/main/java/io/helidon/examples/data/mysql/PokemonService.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.examples.data.mysql;
+
+import io.helidon.http.BadRequestException;
+import io.helidon.service.registry.Services;
+import io.helidon.transaction.Tx;
+import io.helidon.webserver.http.Handler;
+import io.helidon.webserver.http.HttpRules;
+import io.helidon.webserver.http.HttpService;
+import io.helidon.webserver.http.ServerRequest;
+import io.helidon.webserver.http.ServerResponse;
+
+class PokemonService implements HttpService {
+
+    // Helidon Data repository interface providing Pokemon entity operations.
```

Comment:
Remove comment.

Reply:
OK, no comments, no explanation to the users what it is and why it's there.